Library/Formula/pacapt.rb

```diff
@@ -0,0 +1,15 @@
+class Pacapt < Formula
+
+  homepage "https://github.com/icy/pacapt" 
+  url "https://github.com/icy/pacapt.git", :revision => "70cae6c4022696decb6ac1f807a00049e589d9d4", :tag => "v2.0.2"
```

Comment:
Interesting.  If you look into the Makefile and compile.sh both use git commands (make for cleaning, which might be irrelevant for brew). While running the compile file it just ended with non zero e it code having the output file empty. But I tested it on zsh on Linux that time.

Since you guys wanted me to use not only the tag, but also a specific revision, it should be better to use git anyway, right? Because this tarball is purely tag/branch dependent ;)



> On 31 Mar 2015, at 00:23, Dominyk Tiller <notifications@github.com> wrote:
> 
> In Library/Formula/pacapt.rb:
> 
> > @@ -0,0 +1,15 @@
> > +class Pacapt < Formula
> > +
> > +  homepage "https://github.com/icy/pacapt" 
> > +  url "https://github.com/icy/pacapt.git", :revision => "70cae6c4022696decb6ac1f807a00049e589d9d4", :tag => "v2.0.2"
> Odd. Can you post the error you're getting? It's working for me:
> 
> ==> Downloading https://github.com/icy/pacapt/archive/v2.0.2.tar.gz
> /usr/bin/curl -fLA Homebrew 0.9.5 (Ruby 2.0.0-481; Mac OS X 10.10.3) https://github.com/icy/pacapt/archive/v2.0.2.tar.gz -C 0 -o /usr/local/var/homebrew/cache/pacapt-2.0.2.tar.gz.incomplete
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
> 100   118    0   118    0     0     92      0 --:--:--  0:00:01 --:--:--    92
> 100 11072  100 11072    0     0   4329      0  0:00:02  0:00:02 --:--:-- 10833
> ==> Verifying pacapt-2.0.2.tar.gz checksum
> tar xf /usr/local/var/homebrew/cache/pacapt-2.0.2.tar.gz
> ==> make install BINDIR=/usr/local/Cellar/pacapt/2.0.2/bin
> ==> Cleaning
> Fixing /usr/local/Cellar/pacapt/2.0.2/bin/pacapt permissions from 755 to 555
> ==> Finishing up
> ln -s ../Cellar/pacapt/2.0.2/bin/pacapt pacapt
> —
> Reply to this email directly or view it on GitHub.
>